### PR TITLE
Feature/gridedit 704 delete block exterior

### PR DESCRIPTION
**Pull request by BillSenior**

### CODE REVIEW

**Review by lucacarniato — APPROVED** [review]

File: libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGridDeleteInterior.hpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// All indications and logos of, and references to, "Delft3D" and "Deltares"
+// are registered trademarks of Stichting Deltares, and remain the property of
+// Stichting Deltares. All rights reserved.
+//
+//------------------------------------------------------------------------------
+
+#pragma once
+
+#include "CurvilinearGridAlgorithm.hpp"
+
```
Comment (BillSenior) [inline libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGridDeleteInterior.hpp:31]: Should the CurvilinearGrid.hpp be included.

File: libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGridDeleteExterior.hpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+// All indications and logos of, and references to, "Delft3D" and "Deltares"
+// are registered trademarks of Stichting Deltares, and remain the property of
+// Stichting Deltares. All rights reserved.
+//
+//------------------------------------------------------------------------------
+
+#pragma once
+
+#include "CurvilinearGridAlgorithm.hpp"
+
```
Comment (BillSenior) [inline libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGridDeleteExterior.hpp:31]: Should CurvilnearGrid.hpp be included

**Review by BillSenior — COMMENTED** [review]
Just two very minor (exactly the same) questions

File: libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGridDeleteExterior.hpp (hunk 2 above)
Comment (lucacarniato) [reply libs/MeshKernel/include/MeshKernel/CurvilinearGrid/CurvilinearGridDeleteExterior.hpp:31]: it is included in CurvilinearGridAlgorithm.hpp

**Pull request merged**